lib/oneops/api.rb

```diff
@@ -1,5 +1,6 @@
-require 'rubygems'
+require 'active_support'

```

Comment:
Re-ordered these deps and added `active_support` to get around ActiveSupport::Autoload errors